doc/src/extensions.rst

```diff
@@ -274,11 +274,17 @@ If some variables have zero probability in all scenarios, then you will need to
 all variable probabilities! So you might want to set this to False to verify that the probabilities sum to one
 only for the Vars you expect before setting it to True.
 
-Scenario_lpwriter
------------------
-
-This extension writes an lp file with the model and json file with (a) list(s) of
-scenario tree node names and nonanticaptive variables for each scenario before
-the iteration zero solve of PH or APH. Note that for two-stage problems, all
-json files will be the same. See ``mpisppy.generic_cylinders.py``
-for an example of use.
+Scenario_lp_mps_writer
+----------------------
+
+This extension writes an lp file and an mps file with the model as well as a
+json file with (a) list(s) of scenario tree node names and
+nonanticaptive variables for each scenario before the iteration zero
+solve of PH or APH. Note that for two-stage problems, all json files
+will be the same. See ``mpisppy.generic_cylinders.py`` for an example
+of use. In that program it is activated with the
+``--scenario-lp-mps-writer`` option. Note that it
```

Comment:
```suggestion
``--scenario-lp-mps-files`` option. Note that it
```

To be consistent with below? But I think having "write" in the name would be better, e.g., ``--write-scenario-lp-mps-files`` so it's clear what direction we are going.

Reply:
Done.